ee/Earthfile

```diff
@@ -1,16 +1,27 @@
 VERSION 0.8
 
-ARG core=github.com/formancehq/earthly
-
-IMPORT $core AS core
+IMPORT github.com/formancehq/earthly:baaa0352c718937be6f9c40fe40879642bbd6132 AS core
 IMPORT .. AS stack
 
 deploy:
-    FROM core+base-image
-    ARG --required components
-    BUILD --pass-args ./$components+deploy
+  FROM core+base-image
+  ARG --required components
+  BUILD --pass-args ./$components+deploy
 
 deploy-staging:
-    FROM core+base-image
-    ARG --required components
-    BUILD --pass-args ./$components+deploy-staging
\ No newline at end of file
+  FROM core+base-image
+  ARG --required components
+  BUILD --pass-args ./$components+deploy-staging
+    
+run:
+  LOCALLY
```

Comment:
I think I have seen earthly allow `BUILD ./*+$TARGET`, can you check?

Reply:
This is not possible because it is not deterministic.